authors/apps/articles/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import ArticleView, ArticleRetrieveUpdateDestroy
+
+urlpatterns = [
+    path('articles/', ArticleView.as_view(), name="articles"),
+    path('articles/<str:slug>/', ArticleRetrieveUpdateDestroy.as_view()),
```

Comment:
Kindly give this path a `name=""` as you did for the one above it

Reply:
cool. let me do that